PagerDuty/pagerduty.30s.rb

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/ruby
+# <bitbar.title>PagerDuty</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.a uthor>Allan Frese</bitbar.author>
+# <bitbar.author.github>frese</bitbar.author.github>
+# <bitbar.desc>Shows current pagerduty alert status.</bitbar.desc>
+# <bitbar.dependencies>ruby (httparty gem)</bitbar.dependencies>
+
+require "json"
+require "pp"
+require "optparse"
+require "httparty"
+require "date"
+
+#--------------------------------------------------------------------
+# Create a configuration file ".pagerduty" in your HOME directory
+# TOKEN: <pagerduty-token>
+# DOMAIN: <your-pagerduty-domain>
+# USERID: <your-pager-duty-user-id>
+#--------------------------------------------------------------------
+
+class PagerDuty
+
+    def main
+        begin
+            # Init some defaults
+            $verbose = false
+            $id      = nil
+            $command = "GET"
+            $color   = Hash.new
+            begin
+                $config  = YAML.load(File.read("#{ENV['HOME']}/.pagerduty"))
```

Comment:
I would be happy to change to environment variables - but it seems very awkward  as to how you define these in various OS X releases. ! 
And right now I can't seem to figure out how to set them in El Capitan ?? I've tried .bash_profile, /etc/launchd.conf, "launchctl setenv ..." and .MacOSX/environment.plist ?
How can I set environment variables that will be available for bitbar plugins ?